.gitignore

```diff
@@ -21,8 +20,9 @@ coverage
 **/*.metadata.json
 *.metadata.json
 /bundles
-/lib/tests
 build/
+lib/
```

Comment:
Having lib at repository allows to install from git.
`npm install jhipster/react-jhipster`
I prefer to keep it.

Reply:
:-(. npm install installs it from the npm registry, doesn't it?

Reply:
It allows to use react-jhipster snapshot without publishing.
Like here https://github.com/qmonmert/generator-jhipster/blob/24360b2003f54a86b92baad2815d3dd9ba3795b8/generators/client/templates/react/package.json#L15.